app/controllers/books_controller.rb

```diff
@@ -31,14 +28,33 @@ def update
   end
 
   def destroy
-    authorize :book
-    book = Book.find(params[:id])
     book.destroy
     head :no_content
   end
 
+  def rate_book
+    if rating && rating.update_attributes(score: params[:score])
```

Comment:
with the new rating method there's no need for testing this, just update the score and return an ok code